assets/app/scripts/services/metrics.js

```diff
@@ -113,13 +123,19 @@ angular.module("openshiftConsole")
         return getMetricsURL().then(function(metricsURL) {
           var reqURL,
               template = metricsURL + templateByMetric[config.metric],
+              containerName = config.containerName,
               buckets = 60,
               start,
               end = config.end || Date.now();
 
+          // network metrics are not available at container level (only at pod and node level)
```

Comment:
We should make that clear in the UI somehow. "Container Metrics" and "Pod Metrics" subheadings? @jwforres opinions?